CRM/Upgrade/Incremental/Base.php

```diff
@@ -220,6 +220,30 @@ public static function addColumn($ctx, $table, $column, $properties, $localizabl
     return TRUE;
   }
 
+  /**
+   * Add the specified option group, gracefully if it already exists.
+   *
+   * @param CRM_Queue_TaskContext $ctx
+   * @param array $params
+   * @param array $options
+   *
+   * @return bool
+   */
+  public static function addOptionGroup(CRM_Queue_TaskContext $ctx, $params, $options): bool {
+    $defaults = [
+      'is_reserved' => 1,
+      'is_active' => 1,
+      'is_locked' => 1,
+    ];
```

Comment:
These defaults seem more like "gotchas" for those not expecting them to be auto-applied :/
I think upgrade functions would be easier to grok if every attribute was explicitly passed in.

Reply:
@colemanw yeah - I left is_active there but moved the other 2 out - seems like you wouldn't create them inactive & if you did that would be pretty intentional